### ISSUE TITLE
[REQ] Custom cases

### ISSUE THREAD

**Issue by RandomCSGO** [ianlucas/cs2-inventory-simulator#88]
Possibility of creating personalized cases with the game items of our choice as well as the percentages of our choice.

**Comment by ianlucas** [comment]
This is currently out of scope for the project, maybe after #25 is resolved reopen this issue so that I can reevaluate it.
